tests/test_uploader.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+
+from ansys.fluent.core import launch_fluent
+
+
+def test_base_session():
+    base_session = launch_fluent(cleanup_on_exit=False, mode="meshing")
+    with pytest.raises(Exception) as e_info:
+        base_session._upload(
+            "D:/PythonPractise/RealWithUnits/ConversionTable.py", "testNow.py"
```

Comment:
Is the path `D:/..` from local machine? This won't accesible from linux docker container. Plase use a path relative to `pyfluent.EXAMPLES_PATH`

Reply:
Ok. Thank you.

Reply:
Done.